.gitmodules

```diff
@@ -46,4 +46,7 @@
 	branch = rustc/8.0-2019-01-16
 [submodule "src/doc/embedded-book"]
 	path = src/doc/embedded-book
-	url = https://github.com/rust-embedded/book.git
\ No newline at end of file
+	url = https://github.com/rust-embedded/book.git
+[submodule "src/parking_lot"]
+	path = src/parking_lot
+	url = https://github.com/faern/parking_lot
```

Comment:
This can be pointed back to @Amanieu's repository after we are done integrating the changes needed to the crate.